src/instance/minecraft/commands/KuudraCommand.ts

```diff
@@ -0,0 +1,46 @@
+import { KuudraTier } from 'hypixel-api-reborn'
+import { ChatCommandContext, ChatCommandHandler } from '../common/ChatInterface'
+
+const Kuudra: Record<KuudraTier, string[]> = {
+  none: ['basic', 't1', '1'],
+  hot: ['hot', 't2', '2'],
+  burning: ['burning', 't3', '3'],
+  fiery: ['fiery', 't4', '4'],
+  infernal: ['infernal', 't5', '5']
+}
+
+export default {
+  triggers: ['kuudra', 'k'],
+  enabled: true,
+  handler: async function (context: ChatCommandContext): Promise<string> {
+    let chosenTier: KuudraTier | undefined
```

Comment:
move that down to where it first used. Only args are put first. put it above the first for loop on line 20.